apps/src/templates/instructions/teacherFeedback/ReviewState.jsx

```diff
@@ -2,10 +2,10 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import i18n from '@cdo/locale';
 import {KeepWorkingBadge} from '@cdo/apps/templates/progress/BubbleBadge';
-import {ReviewStates} from '@cdo/apps/templates/types';
+import {ReviewStates} from '@cdo/apps/templates/instructions/teacherFeedback/types';
 import color from '@cdo/apps/util/color';
 
-class FeedbackStudentReviewState extends React.Component {
+class ReviewState extends React.Component {
```

Comment:
(minor) `Maybe call this one ReadOnlyReviewState` for symmetry.  Or maybe `ReviewStateTeacher` and `ReviewStateStudent` would be clearer?